Index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>FREECODECAMP SURVEY FORM</title>
+    <link rel="stylesheet" href="CSS.css">
+</head>
+<body>
+    <h1 id="title">FREECODECAMP SURVEY FORM</h1>
+    <p id="description">THANK YOU FOR TAKING THE TIME TO HELP US IMPROVE THE PLATFORM</p>
+    <form id="survey-form">
+        <div class="input-element">
+            <label id="name-label">NAME</label>
+            <input required placeholder="Name" type="text" id="name">
+        </div>
+
+        <div class="input-element">
+            <label id="email-label">EMAIL</label>
+            <input required placeholder="Email" type="email" id="email">
+        </div>
+
+        <div class="input-element">
+            <label id="number-label">NUMBER</label>
+            <input placeholder="Number" type="number" min="3" max="10" id="number">
+
+        </div>
+
+        <div class="input-element">
```

Comment:
(OPTIONAL)
Given the simplicity of your code, I recommend using the section tag instead of div tags for lines 30, 41, 61, and 139. This makes it easier to see when a new main container begins and ends.